projects/plugins/jetpack/extensions/blocks/ai-image/index.js

```diff
@@ -30,6 +30,12 @@ export const settings = {
 					'jetpack'
 				) }
 			</p>
+			<p>
```

Comment:
Wondering if all or part of this paragraph should be highlighted (`<strong>`, for example) to make it stand out more?